src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -2035,11 +2030,15 @@ fn install_llvm_file(
         // If we have a symlink like libLLVM-18.so -> libLLVM.so.18.1, install the target of the
         // symlink, which is what will actually get loaded at runtime.
         builder.install(&t!(fs::canonicalize(source)), destination, 0o644);
-        if install_symlink {
-            // If requested, also install the symlink. This is used by download-ci-llvm.
-            let full_dest = destination.join(source.file_name().unwrap());
-            builder.copy(&source, &full_dest);
-        }
+
+        // Replace the symlink itself with an equivalent linker script. This is used by
+        // download-ci-llvm and projects linking against librustc_driver.so. We don't use a
+        // symlink, as these are not allowed inside rustup components.
+        let link = t!(fs::read_link(source));
```

Comment:
This will be a relative path, right?

Reply:
Yes, this will produce something like `INPUT(libLLVM.so.18.1-rust-dev)`.